keps/sig-storage/177-volume-snapshot/README.md

```diff
@@ -231,24 +251,305 @@ Here are the changes since the original design proposal:
+    - Usage description:
+      - Impact of its outage on the feature: installation of snapshot CRDs, snapshot-controller, and CSI external-snapshotter sidecar are required for the volume snapshot feature to work. Snapshot validation webhook is optional. If the validation webhook is not running, API validation will not happen which means there may be invalid snapshot API objects being created.
+      - Impact of its degraded performance or high-error rates on the feature:
+
+
+### Scalability
+
+_For alpha, this section is encouraged: reviewers should consider these questions
+and attempt to answer them._
+
+_For beta, this section is required: reviewers must answer these questions._
+
+_For GA, this section is required: approvers should be able to confirm the
+previous answers based on experience in the field._
+
+* **Will enabling / using this feature result in any new API calls?**
+  Describe them, providing:
+  - API call type (e.g. PATCH pods)
+
+```
+    - apiGroups: ["snapshot.storage.k8s.io"]
+      resources: ["volumesnapshotclasses"]
+      verbs: ["get", "list", "watch"]
+    - apiGroups: ["snapshot.storage.k8s.io"]
+      resources: ["volumesnapshotcontents"]
+      verbs: ["create", "get", "list", "watch", "update", "delete"]
+    - apiGroups: ["snapshot.storage.k8s.io"]
+      resources: ["volumesnapshots"]
+      verbs: ["get", "list", "watch", "update"]
+    - apiGroups: ["snapshot.storage.k8s.io"]
+      resources: ["volumesnapshots/status"]
+      verbs: ["update"]
+    - apiGroups: ["snapshot.storage.k8s.io"]
+      resources: ["volumesnapshotcontents/status"]
+      verbs: ["update"]
+```
+
+  - estimated throughput
+  - originating component(s) (e.g. Kubelet, Feature-X-controller)
+  focusing mostly on:
+  - components listing and/or watching resources they didn't before
+    snapshot-controller and CSI external-snapshotter sidecar
+  - API calls that may be triggered by changes of some Kubernetes resources
+    (e.g. update of object X triggers new updates of object Y)
+  - periodic API calls to reconcile state (e.g. periodic fetching state,
+    heartbeats, leader election, etc.)
+    leader election
+
+* **Will enabling / using this feature result in introducing new API types?**
+  Describe them, providing:
+  - API type: volumesnapshots, volumesnapshotcontents, and volumesnapshotclasses are CRDs
+  - Supported number of objects per cluster
+  - Supported number of objects per namespace (for namespace-scoped objects)
+
+* **Will enabling / using this feature result in any new calls to the cloud
+provider?**
+  If a cloud provider has implemented CSI snapshot functionality, it will be called when snapshot operations are triggered.
+
+* **Will enabling / using this feature result in increasing size or count of
+the existing API objects?**
+  Describe them, providing:
+  - API type(s): New CRDs volumesnapshots, volumesnapshotcontents, and volumesnapshotclasses are added
```

Comment:
pvc object also increases because of datasource

Reply:
Done